### PR TITLE
CI: chache trunk and wasm-pack binaries

### PR DESCRIPTION
**Pull request by flosse**

### CODE REVIEW

**Review by uklotzde — CHANGES_REQUESTED** [review]
Why is this needed?

The contents of `~/.cargo` [should already be cached](https://github.com/Swatinem/rust-cache#cache-details) and the runs confirm this: https://github.com/kartevonmorgen/openfairdb/actions/runs/2694377012. Installing Trunk finishes in 1 sec.

File: .github/workflows/continuous-integration.yaml [hunk 1]
```diff
@@ -59,9 +59,21 @@ jobs:
...
+            ~/.cargo/bin/trunk*
+            ~/.cargo/bin/wasm-pack*
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+
+      - name: Install wasm-pack
+        if: steps.cache-cargo-bins.outputs.cache-hit != 'true'
+        run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
+
+      - name: Install Trunk
+        if: steps.cache-cargo-bins.outputs.cache-hit != 'true'
```
Comment (uklotzde) [inline .github/workflows/continuous-integration.yaml:75]: Not needed. `cargo install` detects this.

**Comment by flosse** [comment]
> Why is this needed?
> 
> The contents of `~/.cargo` [should already be cached](https://github.com/Swatinem/rust-cache#cache-details) and the runs confirm this: https://github.com/kartevonmorgen/openfairdb/actions/runs/2694377012. Installing Trunk finishes in 1 sec.

In https://github.com/kartevonmorgen/openfairdb/runs/7413358345 it took 8 minutes :face_with_spiral_eyes: but probably this is because a new Rust version was released and I didn't noticed it :see_no_evil:

**Comment by uklotzde** [comment]
If we need wasm-pack then we could simply use `cargo install wasm-pack` and let Cargo take care of the version check. Same as for Trunk.

**Pull request closed without merging**